styles/header.scss

```diff
@@ -0,0 +1,29 @@
+.nsdc-header{
+  background-color: #fff;
+}
+
+.header-link:hover{
+  border-bottom: 3px solid black;
+}
+
+.border-banner {
+  background-image: url(https://assets.hexcdn.uk/nsdc/assets/img/border-banner.svg);
```

Comment:
Can we guarantee that this image hosted by a 3rd party will always be available?

Reply:
I've got a local copy of it now